data/reference/Instruments.json

```diff
@@ -453,5 +453,12 @@
         "telescope": "JWST",
         "description": "Fixed Slit",
         "reference": null
+    },
+    {
+        "instrument": "IRC",
+        "mode": "NIR",
+        "telescope": "AKARI",
+        "description": null,
+        "reference": "Mura07"
```

Comment:
Use this publication as the reference for the IRC camera: [10.1093/pasj/59.sp2.S401](https://ui.adsabs.harvard.edu/link_gateway/2007PASJ...59S.401O/doi:10.1093/pasj/59.sp2.S401).